server/src/main/java/com/cloud/api/query/dao/UserVmJoinDaoImpl.java

```diff
@@ -344,9 +344,9 @@ public UserVmResponse newUserVmResponse(ResponseView view, String objectName, Us
                 userVmResponse.setPoolType(userVm.getPoolType().toString());
             }
 
-            // Remove blacklisted settings if user is not admin
+            // Remove deny listed settings if user is not admin
             if (caller.getType() != Account.ACCOUNT_TYPE_ADMIN) {
-                String[] userVmSettingsToHide = QueryService.UserVMBlacklistedDetails.value().split(",");
+                String[] userVmSettingsToHide = QueryService.UserVMDenyListedDetails.value().split(",");
```

Comment:
Let's rename it simply to `UserVmDenyListDetails`?

Reply:
> Let's rename it simply to `UserVmDenyListDetails`?

renamed to 'UserVMDeniedDetails' (in sync with config name).